src/components/common.js

```diff
@@ -351,10 +351,17 @@ export const AsyncCreatableSelect = props => {
   return h(RAsyncCreatableSelect, _.merge(commonSelectProps, props))
 }
 
-export const PageBox = ({ children, style = {}, ...props }) => {
+export const PageBoxVariants = {
+  LIGHT: 'light'
+}
+
+export const PageBox = ({ children, variant = undefined, style = {}, ...props }) => {
   return div(_.merge({
     style: {
-      margin: '1.5rem', padding: '1.5rem 1.5rem 0', minHeight: 125, flex: 'none', zIndex: 0, ...style
+      margin: '1.5rem', padding: '1.5rem 1.5rem 0', minHeight: 125, flex: 'none', zIndex: 0,
+      ...Utils.switchCase(variant,
+        [PageBoxVariants.LIGHT, () => { return { backgroundColor: colors.light(), margin: 0, padding: '3rem 3rem 1.5rem' } }],
+        [Utils.DEFAULT, () => {}]), ...style
```

Comment:
This currently actually returns `undefined`, instead of a literal object.
```suggestion
        [Utils.DEFAULT, () => ({})]), ...style
```